eng/testing/tests.mobile.targets

```
@@ -33,6 +33,17 @@
     <UseNativeHttpHandler>true</UseNativeHttpHandler>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetOS)' == 'iOS' and '$(UseDefaultiOSFeatureSwitches)' != 'false'">
```

nit: in the msbuild world we usually condition something on either being `true` or not:

```suggestion
  <PropertyGroup Condition="'$(TargetOS)' == 'iOS' and '$(UseDefaultiOSFeatureSwitches)' == 'true'">
```

---

I agree that is a better practice however it seems like in this case they are trying to achieve having default switches that are enabled (when they are not set to false). It seems like the mobile testing infra already has more switches already with this behavior... `UseDefaultAndroidFeatureSwitches`, `UseDefaultBlazorWASMFeatureSwitches`, etc. In order for this to work a default to `true` would need to be added in the case when this switches are not set, which I agree is a better practice. So if we update this switch, I think we should do it for all others as well.

---

Addressed, thanks.